<?xml version="1.0"?>
<launch>
	<!-- Converting MarkerArray to safe message type SafeObstacleArray.  -->
	<node type="VisualizationMarkerArray2MsgSafe.py" name="marker_array_2_msg_safe" pkg="msg_safe" respawn="true" output="screen">
		<param name="topicVisualizeIn" value="/yolo/BBox3d/Multisense" />
	</node>
</launch>
